lib/sw-precache.js

```diff
@@ -116,6 +116,9 @@ function stringifyToolboxOptions(options) {
   var str = JSON.stringify(options);
   if (options.origin instanceof RegExp) {
     str = str.replace(/("origin":)\{\}/, '$1' + options.origin);
+  } else if (options.successResponses instanceof RegExp) {
```

Comment:
I think this should be just a standalone `if (...)` rather than an `else if (...)`, since someone might set both `options.origin` and `options.successResponses` to `RegExp`s.